src/internal/analytics/__tests__/components/analytics-funnel.test.tsx

```diff
@@ -332,10 +332,29 @@ describe('AnalyticsFunnelStep', () => {
     expect(FunnelMetrics.funnelStepStart).toHaveBeenCalledTimes(1);
     expect(FunnelMetrics.funnelStepStart).toHaveBeenCalledWith({
       funnelInteractionId: mockedFunnelInteractionId,
+      stepName: undefined,
```

Comment:
Why stepName is undefined here? Is there a real difference between this value being an empty string and undefined?

Reply:
I've updated the code in the next revision to have a proper step name